gateway/serve_http_content.go

```diff
@@ -475,9 +475,18 @@ func (i *handler) seekToStartOfFirstRange(w http.ResponseWriter, r *http.Request
 	return true
 }
 
-func seekToRangeStart(data io.Seeker, ra *ByteRange) error {
+func seekToRangeStart(data io.Seeker, ra *ByteRange, size int64) error {
```

Comment:
We should figure out how much testing is worth doing here vs just adding to gateway conformance and relying on that.

It's notable that there was a TODO around testing this area 🤦 https://github.com/ipfs/boxo/pull/922/files#diff-5b9dda4053dba85f902d2464389f79cdfca0976873fd206acaefcb09784d6e98R55, let's drop that TODO and replace with enough testing that we're happy 😅.

Reply:
Dropped todo + added e2e tests to https://github.com/ipfs/gateway-conformance/pull/213, this way the same tests will be applied to inbrowser.link.